_data/mentors.yml

```diff
@@ -45,3 +45,19 @@
   twitter: jigyasa_grover
   facebook: jigyasa.grover.14
   bio:
+  
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*SpaceConsistencyBear, severity NORMAL, section `default`.*

The issue can be fixed by applying the following patch:

```diff
--- a/_data/mentors.yml
+++ b/_data/mentors.yml
@@ -45,7 +45,7 @@
   twitter: jigyasa_grover
   facebook: jigyasa.grover.14
   bio:
-  
+
 - name: Damini Satya
   github: daminisatya
   image: daminisatya.jpg
```